src/proxy/processors/push-action/index.js

```diff
@@ -11,3 +11,4 @@ exports.checkCommitMessages = require('./checkCommitMessages').exec;
 exports.checkAuthorEmails = require('./checkAuthorEmails').exec;
 exports.checkUserPushPermission = require('./checkUserPushPermission').exec;
 exports.clearBareClone = require('./clearBareClone').exec;
+exports.checkSensitiveData = require('./checkSensitiveData').exec;
```

Comment:
Add missing newline at the end of the file.
```suggestion
exports.checkSensitiveData = require('./checkSensitiveData').exec;

```